src/test/kotlin/no/nav/klage/oppgave/domain/klage/KvalitetsvurderingTest.kt

```diff
@@ -0,0 +1,46 @@
+package no.nav.klage.oppgave.domain.klage
+
+import no.nav.klage.oppgave.domain.kodeverk.KvalitetsavvikOversendelsesbrev
+import no.nav.klage.oppgave.exceptions.ValidationException
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+internal class KvalitetsvurderingTest {
+
+    @Test
+    fun `riktig utfylt kvalitetsvurdering passerer`() {
+        val kvalitetsvurdering = Kvalitetsvurdering(
+            oversendelsesbrevBra = true,
+            utredningBra = true,
+            vedtakBra = true
+        )
+        kvalitetsvurdering.validate()
+        assert(true)
```

Comment:
Dette er ikke `assert` fra et testrammeverk uten en innbygget funksjon i Java/Kotlin som kan være i bruk eller ikke, litt basert på config av JVM. Bør ikke brukes i unit-tester. Jeg tenker heller ikke at du trenger en assert. Hvis exception kastes så feiler testen, eller ikke. Du kan ev. utvide testbeskrivelsen med "does not throw exception" e.l.